src/components/tabular-view.js

```diff
@@ -105,14 +141,53 @@ class TabularView extends Component {
                       this.setState({ showDeleteModal: false })
                     }}
                     handleSubmit={() => {
-                      this.handleDelete(folder.id)
+                      this.handleDelete()
                     }}
-                    type="remove"
-                    item="file"
+                    type='remove'
+                    item='file'
                   />
                 </Table.Cell>
               </Table.Row>
             ))}
+
+          {currentFolder &&
+            currentFolder.files &&
+            currentFolder.files.map((file, index) => (
+              <Table.Row
+                key={index}
+                active={activeItems.some(elem => elem.obj.id == file.id)}
+                styleName='index.table-row'
+                onClick={this.handleFileClick(file)}
+              >
+                <Table.Cell>
+                  <Icon size='large' name='file' color={getTheme()} />
+                  {file.fileName}
+                </Table.Cell>
+                <Table.Cell>
+                  {getModifiedDate(file.datetimeModified)}
+                </Table.Cell>
+                <Table.Cell>{file.permission}</Table.Cell>
+                <Table.Cell>
+                  <PopupView
+                    id={file.id}
+                    options={options}
+                    handleOptions={this.handleOptions}
+                  />
+                </Table.Cell>
+              </Table.Row>
+            ))}
+          <ConfirmModal
```

Comment:
combine this modal and modal in folder row and combine it and take outside of the Table as inside the loop it also gets multiplied and crowds the DOM